src/WebJobs.Script/Description/ProxyMetadata.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    public class ProxyMetadata : FunctionMetadata
```

Comment:
Resolved.

---
In reply to: [132557806](https://github.com/Azure/azure-webjobs-sdk-script/pull/1749#discussion_r132557806) [](ancestors = 132557806)